Scoreboard handle in daemon mode should be set to NULL for Apache 2.4 to avoid
crash in lingering close.

https://code.google.com/p/modwsgi/source/detail?path=/mod_wsgi.c&name=mod_wsgi-3.X&r=bdbeacb88f34
--- a/mod_wsgi.c
+++ b/mod_wsgi.c
@@ -10600,7 +10600,13 @@ static void wsgi_process_socket(apr_pool_t *p, apr_socket_t *sock,
      * will add their own input/output filters to the chain.
      */
 
+#if AP_MODULE_MAGIC_AT_LEAST(20110619,0)
+    /* For 2.4 a NULL sbh pointer should work. */
+    sbh = NULL;
+#else
+    /* For 2.2 a dummy sbh pointer is needed. */
     ap_create_sb_handle(&sbh, p, -1, 0);
+#endif
 
     c = (conn_rec *)apr_pcalloc(p, sizeof(conn_rec));
 
